client/state/login/reducer.js

```diff
@@ -57,6 +62,12 @@ export const twoFactorAuthRequestError = createReducer( null, {
 	[ TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST_FAILURE ]: ( state, { error } ) => error
 } );
 
+export const twoFactorAuthPushPoll = createReducer( { inProgress: false, success: false }, {
```

Comment:
I'm not sure we should use `createReducer` here as I believe it persists data by default. We probably want to reset to the default values on reload.